classes/battle/BattleManager.php

```diff
@@ -416,14 +416,18 @@ private function parseCombatText(string $text, Fighter $attacker, Fighter $targe
             [
                 '[player]',
                 '[opponent]',
-                '[gender]',
-                '[gender2]'
+                '[attackerGender]',
+                '[attackerGender2]',
```

Comment:
```suggestion
                '[gender]',
                '[gender2]',
```
A lot of the jutsu text use `[gender]` for the caster, we'd have to replace all of them, so this is the easy way to ensure backwards compatibility while also allowing for specifying target's gender